cmd/boulder-ca/main.go

```diff
@@ -196,6 +196,17 @@ func main() {
 		}()
 		grpcSrv = s
 	}
+	if c.CA.GRPCOCSPGenerator != nil {
+		s, l, err := bgrpc.NewServer(c.CA.GRPCOCSPGenerator, scope)
+		cmd.FailOnError(err, "Unable to setup CA gRPC server")
+		caWrapper := bgrpc.NewCertificateAuthorityServer(cai)
+		caPB.RegisterOCSPGeneratorServer(s, caWrapper)
+		go func() {
+			err = s.Serve(l)
+			cmd.FailOnError(err, "OCSPGenerator gRPC service failed")
+		}()
+		grpcSrv = s
```

Comment:
If both the `c.CA.GRPCA` service and the `c.CA.GRPCOCSPGenerator` are present it looks like `grpcSrv` will get squashed by the OCSP generator server. I think this means that the cleanup in the `CatchSingals` handler on L214 won't `GracefulStop` the CA GRPC server.